tests/Test_1dtube_cxx/Dockerfile

```diff
@@ -0,0 +1,23 @@
+ARG from=precice/precice-ubuntu1604.home-develop
+FROM $from
+USER root
+
+RUN apt-get -qq update && apt-get -qq install \
+    liblapack-dev && \
+    rm -rf /var/lib/apt/lists/*
+
+
+ARG CACHEBUST
+
+RUN mkdir -p /Output/Postproc
+
+
+WORKDIR /home/precice/
+RUN git clone https://github.com/Eder-K/elastictube1d.git
```

Comment:
This test relies on the changes made to the tutorials structure in [this PR](https://github.com/precice/elastictube1d/pull/16). These are not implemented at this time, so for testing purposes I used my personal fork. This needs to be adjusted when the elastictube PR is merged.